### PR TITLE
feat: Onboard Open Targets Genetics dataset 

### PR DESCRIPTION
**Pull request by gkodukula**
## Description

Pipelines included:

- open_targets:genetics

## Checklist

Note: If an item applies to you, all of its sub-items must be fulfilled

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved
- [x] I'm adding or editing a dataset
  - [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
  - [x] I put all my code inside  `open_targets/genetics` and nothing outside of that directory
- [x] I'm refactoring or cleaning up some code

### CODE REVIEW

File: datasets/open_targets_genetics/pipelines/dataset.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  description: ~
+  dataset_sources: ~
+  terms_of_use: ~
+resources:
+
+  # - type: storage_bucket
+  #   name: "open-targets-genetics"
+  #   uniform_bucket_level_access: True
+  #   location: US
+
```
Comment (adlersantos) [inline datasets/open_targets_genetics/pipelines/dataset.yaml:27]: You won't need these commented lines. You can remove them.

File: datasets/open_targets_genetics/pipelines/_images/copy_bq_datasets/script.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+    pass
+
+
+def main(
+    source_project_id: str,
+    target_project_id: str,
+    service_account: str,
+    dataset_name: str,
+    dataset_versions: typing.List[str],
```
Comment (adlersantos) [inline datasets/open_targets_genetics/pipelines/_images/copy_bq_datasets/script.py:40]: Take a look at my comments on https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/313 and apply them in this script.

**Review by adlersantos — CHANGES_REQUESTED** [review]

File: datasets/open_targets_genetics/pipelines/_images/copy_bq_datasets/script.py (hunk 2 above)
Comment (gkodukula) [reply datasets/open_targets_genetics/pipelines/_images/copy_bq_datasets/script.py]: @adlersantos done

File: datasets/open_targets_genetics/pipelines/dataset.yaml (hunk 1 above)
Comment (gkodukula) [reply datasets/open_targets_genetics/pipelines/dataset.yaml:27]: @adlersantos done

**Review by adlersantos — APPROVED** [review]

**Review by nlarge-google — APPROVED** [review]

**Pull request merged**